explorer/src/Pos/Explorer/ExtraContext.hs

```diff
@@ -41,4 +41,4 @@ makeExtraCtx :: HasConfiguration => ExtraContext
 makeExtraCtx =
     let addressCoinPairs = utxoToAddressCoinPairs $ unGenesisUtxo genesisUtxo
         redeemOnly = filter (isRedeemAddress . fst) addressCoinPairs
-    in ExtraContext $ V.fromList redeemOnly
+    in ExtraContext $ V.fromList redeemOnly
```

Comment:
Please avoid removing newlines at ends of files in particular, and such hidden changes in large PRs in general.